### PR TITLE
feat: paid_event_ticket_counts

### LINKED ISSUE #79: Paid Event ticket Counts - Contract Cairo
Implement the function that calculate the total number of paid tickets purchased for a specific event. 

       **paid_event_ticket_count: Map<u256, u256>** // map<event_id, count_number_of_ticket>

`fn paid_event_ticket_counts(self: @TContractState) -> u256;`

**TEST**
write a unit test for the function and all the edge cases.

**DURATION: 24HRS**

**NOTE
Only Application During ODBoost Period will be considered**

### PR DESCRIPTION
**Pull request by Kaminar-i**
Function that calculate the total number of paid tickets purchased for a specific event.

fixes #79

### CODE REVIEW

File: src/events/chainevents.cairo [hunk 1]
```diff
@@ -284,14 +284,17 @@ pub mod ChainEvents {
             self.attendee_event_registration_counts.read(event_id)
         }
 
-        fn pay_for_event(ref self: ContractState, event_id: u256) {}
+        fn pay_for_event(ref self: ContractState, event_id: u256) {
+            let prev_num_of_paid_tickets = self.paid_event_ticket_count.read(event_id);
```
Comment (mubarak23) [inline src/events/chainevents.cairo:288]: please let remove this, someone is working on this function

he will definitely update the storage once they pay for the event

**Review by mubarak23 — CHANGES_REQUESTED** [review]

**Comment by mubarak23** [comment]
@Kaminar-i sync your fork to get the latest changes that were push to main branch

File: src/events/chainevents.cairo (hunk 1 above)
Comment (mubarak23) [reply src/events/chainevents.cairo:288]: @Kaminar-i the pay for event function PR has been merged, you can sync your fork and write a test for your implementations

**Comment by Kaminar-i** [comment]
Fixed merge conflicts ser @mubarak23
